admin/bin/qserv-configure.py

```diff
@@ -111,22 +112,20 @@ def __init__(self):
         # Defining option of each configuration step
         init_group = parser.add_argument_group('Initialization',
                                                'Creation of QSERV_RUN_DIR and QSERV_DATA_DIR')
-        config_group = parser.add_argument_group('Configuration steps', 'General configuration steps')
-        nodb_component_group = parser.add_argument_group('Components configuration',
-                                                         'Configuration of external components')
-        db_component_group = parser.add_argument_group('Database components configuration',
-                                                       'Configuration of external components '
-                                                       'impacting data,\n'
-                                                       'launched if and only if QSERV_DATA_DIR is empty')
+        config_group = parser.add_argument_group('Configuration steps',
+                                                 'Configuration steps which can be run in standalone mode, '
+                                                 'if none is provided\nthen whole configuration procedure will'
+                                                 ' be launched')
+
         for step_name in configure.ALL_STEPS:
             if step_name in configure.INIT:
                 group = init_group
-            elif step_name in configure.DB_COMPONENTS:
-                group = db_component_group
-            elif step_name in configure.NODB_COMPONENTS:
-                group = nodb_component_group
-            else:
+            # complex steps are removed from options
+            elif step_name in configure.ETC + configure.CLIENT:
                 group = config_group
+            else:
+                continue
+
```

Comment:
-R option must be used on each step if needed, -D option should only be used at initialization step. That's why they're not in the same group.